.github/workflows/code-style-checking.yml

```diff
@@ -1,4 +1,4 @@
-name: Code quality checks
+name: Code style checking
```

Comment:
It isn't really `style checking` only ruff does that...

Reply:
![image](https://github.com/Gwardia-Czapli/GwardiaHub/assets/88268063/299f5ac8-14e4-4f4a-bac7-ca6b8fc8e4f8)

Reply:
I suggested renaming the action responsible for style checking, not renaming it to 'Code style checking'...